### PR TITLE
[IMP] mail: remove legacy popover from CallParticipantCard

### PR DESCRIPTION
**Pull request by zel-odoo**
task-2968161

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/models/call_participant_card_popover_content_view.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        _computeHasConnectionInfo() {
+            return Boolean(this.callParticipantCard.rtcSession && this.env.debug && this.callParticipantCard.channelMember.channel.thread.rtc);
+        },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeInboundConnectionTypeText() {
+            if (!this.callParticipantCard.rtcSession || !this.callParticipantCard.rtcSession.remoteCandidateType) {
+                return sprintf(this.env._t('From %s: no connection'), this.callParticipantCard.channelMember.persona.name);
```
Comment (alexkuhn) [inline addons/mail/static/src/models/call_participant_card_popover_content_view.js:30]: format:
```js
if (
   A ||
   B
) {
```

File: addons/mail/static/src/components/call_participant_card_popover_content_view/call_participant_card_popover_content_view.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component } = owl;
+
+export class CallParticipantCardPopoverContentView extends Component {
+
+    //--------------------------------------------------------------------------
+    // Getters / Setters
+    //--------------------------------------------------------------------------
+
```
Comment (alexkuhn) [inline addons/mail/static/src/components/call_participant_card_popover_content_view/call_participant_card_popover_content_view.js:12]: Would remove comment (useless noise)

File: addons/mail/static/src/components/call_participant_card_popover_content_view/call_participant_card_popover_content_view.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    /**
+     * @returns {CallParticipantCardPopoverContentView|undefined}
```
Comment (alexkuhn) [inline addons/mail/static/src/components/call_participant_card_popover_content_view/call_participant_card_popover_content_view.js:14]: no `undefined` (it doesn't happen anymore)

File: addons/mail/static/src/components/call_participant_card_popover_content_view/call_participant_card_popover_content_view.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.CallParticipantCardPopoverContentView" owl="1">
+        <div class="d-flex flex-column p-3">
+            <input type="range" min="0.0" max="1" step="0.01" t-att-value="callParticipantCardPopoverContentView.callParticipantCard.rtcSession.volume" t-on-change="callParticipantCardPopoverContentView.onChangeVolume"/>
+            <t t-if="callParticipantCardPopoverContentView.hasConnectionInfo">
+                <hr class="o_CallParticipantCardPopoverContentView_volumeMenuAnchor_separator w-100 border-top"/>
```
Comment (alexkuhn) [inline addons/mail/static/src/components/call_participant_card_popover_content_view/call_participant_card_popover_content_view.xml:7]: `o_CallParticipantCardPopoverContentView_volumeMenuAnchor_separator`
=>
`o_CallParticipantCardPopoverContentView_volumeMenuAnchorSeparator`

File: addons/mail/static/src/models/popover_view.js [hunk 5]
```diff
@@ -188,6 +206,15 @@ registerModel({
             compute: '_computeAnchorRef',
             required: true,
         }),
+        callParticipantCardPopoverContentView: one('CallParticipantCardPopoverContentView', {
+            compute: '_computeCallParticipantCardPopoverContentView',
+            inverse: 'popoverViewOwner',
+            isCausal: true,
+        }),
+        callParticipantCardOwner: one('CallParticipantCard', {
```
Comment (alexkuhn) [inline addons/mail/static/src/models/popover_view.js:214]: alphabetical order (`callParticipantCardOwner` before `callParticipantCardPopoverContentView`)

**Review by alexkuhn — COMMENTED** [review]
LGTM otherwise

File: addons/mail/static/src/models/call_participant_card_popover_content_view.js (hunk 1 above)
Comment (zel-odoo) [reply addons/mail/static/src/models/call_participant_card_popover_content_view.js:30]: you should zoom out a little bit, it happens on github sometimes....

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**